src/Bluejay.asm

```diff
@@ -1082,16 +1079,13 @@ exit_run_mode_is_stall:
 
 exit_run_mode_is_stall_beep:
     ; Stalled too many times
-    clr  IE_EA
+    ;clr  IE_EA                         ; Interrupts already disabled above
```

Comment:
I'd remove this line completely, maybe just move the comment up that interrupts are already disabled at this points.

Reply:
Removed